packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -227,7 +230,9 @@ class PlatformViewsService {
       );
     }
     await SystemChannels.platform_views.invokeMethod<void>('create', args);
-    return UiKitViewController._(id, layoutDirection);
+    final UiKitViewController controller = UiKitViewController._(id, layoutDirection);
+    _instance._focusCallbacks[id] = onFocus ?? () {};
```

Comment:
How about only adding if it's non-null instead of an empty block?
```suggestion
    if (onFocus != null) {
      _instance._focusCallbacks[id] = onFocus;
    }
```